apps/cmp.c

```diff
@@ -2608,15 +2608,15 @@ static int get_opts(int argc, char **argv)
             break;
         }
     }
+
+    /* No extra args. */
     argc = opt_num_rest();
     argv = opt_rest();
-    if (argc != 0) {
-        CMP_err1("unknown parameter %s", argv[0]);
-        goto opt_err;
-    }
-    return 1;
+    if (argc == 0)
+        return 1;
 
- opt_err:
+    CMP_err1("unknown parameter %s", argv[0]);
```

Comment:
Not sure if this is better or worse than all the other apps that dont do this format..
get_opts:apps/cmp.c:2618:CMP error: unknown parameter zoomba
get_opts:apps/cmp.c:2620:CMP error: use -help for summary of 'cmp' options

Reply:
I removed a bunch of other "extra" outputs, I missed this one.  Fixup coming.